python/src/block-event-loop/activity.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+from temporalio import activity
+
+from test_types import ComposeGreetingInput
+
+@activity.defn
+def compose_greeting(input_data: ComposeGreetingInput) -> str:
+
+    start_time = time.time()
+
+    for i in range(5):
+        # Simulate some work
+       time.sleep(0.1)
```

Comment:
Prob note that this is a blocking asyncio call?

Reply:
ack